build/debugger.m4

```diff
@@ -1,9 +1,9 @@
 AC_DEFUN([GP_CHECK_DEBUGGER],
 [
     GP_ARG_DISABLE([Debugger], [auto])
-    GP_CHECK_PLUGIN_GTK2_ONLY([Debugger])
+    GP_CHECK_PLUGIN_GTK3_ONLY([Debugger])
```

Comment:
I didn't check how hard it'd be to support both GTK2 and GTK3, but it seems a little sad to switch from one restriction to the other.  Well, I guess it becomes less and less mandatory to support GTK2, but still.